pkg/clientconn/roxctl_command_header.go

```diff
@@ -0,0 +1,10 @@
+package clientconn
+
+// RoxctlCommandHeader is the HTTP header name with the reconstructed
+// roxctl command as the value.
+const RoxctlCommandHeader = "Rh-Roxctl-Command"
```

Comment:
"Rh-" as in "Red Hat"?

Reply:
Yes. I saw headers like `X-Rh-Insights-Request-Id` being used, and decided to keep only `Rh-`, as `X-` is not recommended to use.